### DISCUSSION TITLE
Foliate v3 keyboard shortcuts

### DISCUSSION THREAD

**Discussion by gotmax23** [johnfactotum/foliate#1233] [Q&A]
There used to be a menu to list all the available keyboard shortcuts in Foliate v2. Is this still there in the GTK 4 rewrite? I cannot seem to find it. Are the keyboard shortcuts listed somewhere else?

**Comment by johnfactotum** [comment]
Currently, no. That is a missing feature in v3.

**Comment by PeterGreb** [comment]
I am new to Foliate, and I like to use keyboard shortcuts, but I have not found a list of these.
Perhaps you could put a simple list on your FAQ.

**Comment by afduggirala** [comment]
Hello everyone,
Is there a keyboard shortcut for Find? I have tried Ctrl-F without avail.
thanks,

**Reply by johnfactotum** [reply]
Yes, Ctrl+F should work. You might need to upgrade to v3.1.1. See #1299

**Comment by rghvdberg** [comment]
extracted this from https://github.com/johnfactotum/foliate/blob/b409dac67c87160162c7af1fa41c7b3a1cc8c4b8/src/book-viewer.js#L679-L707

| Key                                                                                  | Action             |
|--------------------------------------------------------------------------------------|--------------------|
| F9                                                                                   | Toggle Sidebar     |
| \<Ctrl-f> /slash                                                                     | Toggle Search      |
| \<Ctrl-l>                                                                            | Show Location      |
| \<Ctrl-i> / <alt>Return                                                              | Show Info          |
| \<Ctrl-t>                                                                            | Toggle Toc         |
| \<Ctrl-Alt-a>                                                                        | Toggle Annotations |
| \<Ctrl-Alt-d> <Ctrl-d>                                                               | Bookmark           |
| \<Alt-comma>                                                                         | Preferences        |
| \<Ctrl-shift-g>                                                                      | Search Prev        |
| \<Ctrl-g>                                                                            | Search Next        |
| \<Ctrl-c>                                                                            | Selection Copy     |
| \<Ctrl-f>                                                                            | Selection Search   |
| F12                                                                                  | Inspector          |
| \<Ctrl-m>                                                                            | Scrolled           |
| \<Ctrl-r>                                                                            | Reload             |
| plus / equal / KP_Add / KP_Equal <br> \<Ctrl-plus> / \<C-equal> / \<C-KP_Add> / \<C-KP_Equal> | Zoom In            |
| minus / KP Subtract / \<Ctrl-minus> / \<C-KP Subtract>                                        | Zoom Out           |
| 0 / 1 / KP 0 / \<Ctrl-0> / \<C-KP 0>                                                   | Zoom Restore       |
| p / Page_Up / \<shift-space>                                                          | Prev               |
| n / Page_Down / space                                                                | Next               |
| k / Up                                                                               | Scroll Up          |
| j / Down                                                                             | Scroll Down        |
| h / Left                                                                             | Go Left            |
| l / Right                                                                            | Go Right           |
| \<Alt-Left>                                                                          | Back               |
| \<Alt-Right>                                                                         | Forward            |
| \<Ctrl-p>                                                                            | Print              |

**Comment by rghvdberg** [comment]
there's also  https://github.com/johnfactotum/foliate/blob/b409dac67c87160162c7af1fa41c7b3a1cc8c4b8/src/app.js#L207-L213     'app.about': ['F1']

**Comment by djinnalexio** [comment]
feat: ✨ Added the keyboard shortcuts help page #1607
